e2etests/node_scan_rhcos_test.go

```diff
@@ -136,16 +134,18 @@ func TestGRPCGetRHCOSNodeVulnerabilities(t *testing.T) {
 						Vulnerabilities: []*v1.Vulnerability{vulnTar},
 					},
 					{
-						Name:            "grep",
-						Version:         "3.1-6.el8.x86_64",
+						Name:    "grep",
```

Comment:
See my `.noarch` suggestion https://github.com/stackrox/scanner/pull/1004#issuecomment-1373488790 . Optionally, up to you.

Reply:
Unfortunately `golang-docs.noarch` returns a vulnerability (the same that can be found for `golang-docs:1.19.2.noarch`).


```
Error Trace:	/Users/prygiels/go/src/github.com/stackrox/scanner/e2etests/node_scan_rhcos_test.go:192
Error:      	"[name:"CVE-2022-23806" description:"DOCUMENTATION: A flaw was found in the elliptic package of the crypto library in golang (...)" metadata_v2:<last_modified_date_time:"2022-10-24T00:00Z" cvss_v3:<vector:"CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:H" score:8.2 exploitability_score:3.9 impact_score:4.2 > > severity:"Moderate" ]" should have 0 item(s), but has 1
Test:       	TestGRPCGetRHCOSNodeVulnerabilities/Selected_vulnerabilities_should_be_returned_by_the_certified_scan
Messages:   	Expected to find 0 vulnerabilities for feature 'golang-docs:.noarch'
```

Reply:
It looks like the CVE is for golang and not for the golang-docs, but this is what our scanner returns 🤷

Reply:
How about these?

```
git-core-doc.noarch
iso-codes.noarch
tzdata.noarch
```

Reply:
Following up in https://srox.slack.com/archives/C04213W844U/p1673027151155069